frontend/src/pages/Groups/GroupsList/GroupsList.jsx

```diff
@@ -0,0 +1,96 @@
+import "./GroupsList.scss";
+import { useQuery, gql } from "@apollo/client";
+import useProfileInfo from "../../../hooks/useProfileInfo";
+import ConnectionErrorMessage from "../../../components/ConnectionErrorMessage/ConnectionErrorMessage";
+import { motion } from "framer-motion";
+
+const GROUP_LIST_QUERY = gql`
+  query getGroups($id: ID!) {
+    users(id: $id) {
+      edges {
+        node {
+          groups {
+            edges {
+              node {
+                name
+                metrics {
+                  artworkCount
+                  memberCount
+                }
+                groupPortfolio {
+                  artworks(first: 4) {
+                    edges {
+                      node {
+                        pictures
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+export default function GroupsList() {
+  let groups = useGroupList();
+  if (groups.length) {
+    groups = new Array(20).fill(groups[0]);
+  }
```

Comment:
This is just for you all to try out a scrolling display. The code duplicates the first group 20x. I'll take it out before we merge.